server/test/auth/ApiAuthenticatorTest.java

```diff
@@ -254,7 +257,10 @@ public void validate_invalidSecret() {
     var rawCredentials = keyId + ":" + "notthesecret";
 
     assertBadCredentialsException(
-        new FakeRequestBuilder().withRawCredentials(rawCredentials).build(),
+        new FakeRequestBuilder()
+            .withRawCredentials(rawCredentials)
```

Comment:
*nit, non-blocking*: It'd be kind of nice to change this method name to match `remoteAddress()` and not have `with` on the front

Reply:
Good suggestion, thank you!